module/VuFind/src/VuFind/Db/Row/AccessToken.php

```diff
@@ -56,13 +57,13 @@ public function __construct($adapter)
     /**
      * Set user ID.
      *
-     * @param ?string $userId User ID
+     * @param ?UserEntityInterface $user User owning token
      *
      * @return AccessTokenEntityInterface
      */
-    public function setUserId(?string $userId): AccessTokenEntityInterface
+    public function setUser(?UserEntityInterface $user): AccessTokenEntityInterface
     {
-        $this->__set('user_id', $userId);
+        $this->__set('user_id', $user === null ? null : $user->getId());
```

Comment:
Value could be replaced with `$user?->getId()`?

Reply:
Good point -- changed!